src/dali/abstract_pair_converter_plugin.py

```diff
@@ -58,6 +60,8 @@
 _STANDARD_WEIGHT: float = 1
 _STANDARD_INCREMENT: float = 1
 
+_CONFIG_FILE_URL: str = "https://github.com/eprbell/dali-rp2/blob/main/docs/configuration_file.md"
```

Comment:
I suggest:
```suggestion
_CONFIG_FILE_DOC_URL: str = "https://github.com/eprbell/dali-rp2/blob/main/docs/configuration_file.md"
```

Reply:
The identifier is still the old one: unresolving the conversation.

Reply:
This is resolved locally, just waiting for your response to the python question before I commit.